### PR TITLE
feat: add pathFinder function to clean paths

### LINKED ISSUE #7: bug: the subcommand `add` creates duplicate entries in TUI.
When the subcommand `add` is run multiple times on the same directory(same absolute path).
It creates multiple entries of it in TUI.

pms add subcmd

### PR DESCRIPTION
**Pull request by sidkhuntia**
Fixes #7 
This commit introduces a new function, `pathFinder`, which takes a path as input and returns a cleaned version of the path. This function uses the `filepath.Clean` function from the `path/filepath` package in Go, which simplifies any `..` or `.` references in the path.
out

### CODE REVIEW

**Comment by sidkhuntia** [comment]
@JammUtkarsh  what do you think where should i use the `pathFinder` func ?

File: cmd/add.go [hunk 1]
```diff
@@ -63,6 +69,7 @@ func pathResolver(path string) string {
 		cobra.CheckErr(utils.AddProject(wd))
 	default:
 		path = wd + string(os.PathSeparator) + path
+		path = pathFinder(path)
 		cobra.CheckErr(utils.AddProject(path))
```
Comment (jammutkarsh) [inline cmd/add.go]: Change this to 
`cobra.CheckErr(utils.AddProject(filepath.Clean(path)))`

**Comment by jammutkarsh** [comment]
Okay, I didn't knew that a function like `filepath.Clean(path)`. So we don't need to create a function and tests around it, since it comes from std lib and it would have been already tested.

You just need to change one like, which I marked and that will all be good.

**Comment by sidkhuntia** [comment]
Do I need to squash the commits also?

**Comment by jammutkarsh** [comment]
Yes, 1 commit for 1 minor fix or feature. The reason I do this(and make you do this) is to have a cleaner(and presentable) commit history.

I read [this](https://dev.to/samuelfaure/how-atomic-git-commits-dramatically-increased-my-productivity-and-will-increase-yours-too-4a84) and started making atomic commits in locally.
If the contribution guidelines requires me to make a single commit for a feature, then I squash them.

**Comment by sidkhuntia** [comment]
I will give it a read, thanks for sharing

**Review by jammutkarsh — APPROVED** [review]

**Pull request merged**